### PR TITLE
404 page

### PR DESCRIPTION
**Pull request by denzilbilson-google**

### CODE REVIEW

File: src/main/webapp/404.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<link rel="stylesheet" href="404_style.css">
+	</head>
+	<body>
+		<div id="main">
+			<h1>Error 404</h1>
+			<p>Either you messed up or we did. Let's not play the blame game ; )</p>
```
Comment (LeonardZ0731) [inline src/main/webapp/404.html:9]: I like this error message 😆

File: src/main/webapp/404.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+			<a href="/"><input type="button" id="button404" value="Go Back Home" /></a>
```
Comment (LeonardZ0731) [inline src/main/webapp/404.html:10]: Personally I would recommend using `<button onclick="window.location.href='/index.jsp';" id="button404">Go Back Home</button>`

File: src/main/webapp/404.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<link rel="stylesheet" href="404_style.css">
```
Comment (claudiatu) [inline src/main/webapp/404.html:4]: Could we add a page title here?

- [ ] add title

File: src/main/webapp/404_style.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+p {
+	font-size: 2em;
+	text-align: center;
+	font-weight: 100;
+}
+
+h1 {
+	text-align: center;
+	font-size: 8em;
+	font-weight: 600;
```
Comment (claudiatu) [inline src/main/webapp/404_style.css:24]: It looks like we're loading in font weights 400, 100, 300 and 500 on line 1, but here we're using 600. Is this intended?

File: src/main/webapp/404.html (hunk 2 above)
Comment (denzilbilson-google) [reply src/main/webapp/404.html:11]: Not familiar with jsp. Why would this be a better approach?

File: src/main/webapp/404.html (hunk 1 above)
Comment (denzilbilson-google) [reply src/main/webapp/404.html:10]: Thanks!

File: src/main/webapp/404.html (hunk 2 above)
Comment (LeonardZ0731) [reply src/main/webapp/404.html:11]: Ah it's actually not jsp, it's just plain html. It's just a button with a onclick function that redirects you to another url, which has the same functionality as your line 10. So there is no need to change.

**Comment by peap** [comment]
No need for my review; now that claudiatu has write access, her approval counts toward the requirement of getting two.

**Pull request merged**